servant/src/Servant/API/ResponseHeaders.hs

```diff
@@ -31,8 +31,8 @@ module Servant.API.ResponseHeaders
     ) where
 
 import           Data.ByteString.Char8       as BS (pack, unlines, init)
-import           Data.ByteString.Conversion  (ToByteString, toByteString',
-                                              FromByteString, fromByteString)
+import           Web.HttpApiData             (ToHttpApiData, toHeader,
+                                             FromHttpApiData, parseHeader)
 import qualified Data.CaseInsensitive        as CI
```

Comment:
There was a comment earlier about just using `(ToHttpApiData(..)...` here, would that be preferable? Or `(ToHttpApiData(toHeader)...` to make it clear it's coming from that class?